compact/ecal/backward_PbWO4.xml

```diff
@@ -80,7 +99,7 @@
     </comment>
     <readout name="EcalEndcapNHits">
       <segmentation type="NoSegmentation" key="sector"/>
-      <id>system:8,sector:4,module:20</id>
+      <id>system:8,sector:4,row:10,column:10,module:20</id>
```

Comment:
256 cell rows/columns should be more than enough
```suggestion
      <id>system:8,sector:4,row:8,column:8</id>
```

Reply:
Thanks for correct :)

Reply:
solve in the last commit